src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -121,8 +91,39 @@ $schedule-btn--height: 51px;
     }
   }
 
-  &__bookmarked-toggle {
+  &__bookmarkedToggle {
```

Comment:
```suggestion
  &__bookmarked-toggle {
```

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23